### PR TITLE
Connect to be

### LINKED ISSUE #41: Set up initial contribution functionality

### PR DESCRIPTION
**Pull request by attom2**
# Pull request checklist
closes #41 , #42, #44 
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- Hard coded buttons on contribution and new Campaign form

## New behavior

- Buttons are now dynamic, they accept inputs from the user and only run if info is filled out.

## Other information

-still need to have dynamic addresses for each individual campaign

### CODE REVIEW

**Review by jpc20 — APPROVED** [review]
Looks great! It seems like the next steps are to make the campaign creation dynamic, finish the contribute form, and add request functionality.

File: src/components/CampaignDetails.vue [hunk 1]
```diff
@@ -7,18 +8,45 @@
     <h3> Minimum Contribution </h3>
     <h1>{{ campaign.minContribution }}</h1>
     <p>There is some information missing and this has no styles </p>
+    <input v-model='contribution' placeholder='Enter your contribution' type="number"/>
+    <button :disabled="!contribution"
+    @click="submitContribution">
+    Submit Contribution </button>
```
Comment (edmdc) [inline src/components/CampaignDetails.vue:14]: Pretty straightforward to disable a button, it seems. Gotta love vue!

File: src/components/CampaignDetails.vue [hunk 2]
```diff
@@ -7,18 +8,45 @@
...
+    const campaignAddress = "0xe8a0980C2B37C2C4FCE45e579301B00CC824bCFc"
+    this.$store.dispatch('fetchCampaign', campaignAddress)
+  },
+  methods: {
+    async submitContribution() {
+      const campaignAddress = "0xe8a0980C2B37C2C4FCE45e579301B00CC824bCFc"
+      const campaignInstance = await ethCampaign.at(campaignAddress);
+      campaignInstance.contribute({ from: this.$store.state.accountNum, value: this.contribution })
+    }
+  },
```
Comment (edmdc) [inline src/components/CampaignDetails.vue:45]: Is this what you mean by dynamic campaign addresses?

**Review by edmdc — APPROVED** [review]

File: src/components/CampaignDetails.vue (hunk 2 above)
Comment (jpc20) [reply src/components/CampaignDetails.vue:45]: Yeah, I believe this address will be different for each campaign.

**Pull request merged**